CoffeeShop.playground/Contents.swift

```diff
@@ -0,0 +1,413 @@
+import Foundation
+
+// MARK: - Location
+struct Location: Equatable {
+    var x = 0.0
+    var y = 0.0
+
+    func distanceTo(_ location: Location) -> Double {
+        return sqrt( pow(self.x - location.x, 2) + pow(self.y - location.y, 2) )
+    }
+}
+
+
+typealias CoffeeShopWithDistance = (coffeeShop: CoffeeShop, distance: Double)
```

Comment:
How do you decide whether to use typealias or struct?

Reply:
The struct is a coffee shop while the typealias is a renamed tuple which contains a coffee shop and a distance to the provided user location